terraform/modules/app/task_definition.tf

```diff
@@ -32,6 +32,7 @@ locals {
 
 module "app_container_definition" {
   source                = "../../modules/container-definition"
+  image                 = "govuk/${var.image_name}:deployed-to-production"
```

Comment:
I guess we can't have "deployed-to-${env}" because the environment name isn't in scope. Fair enough :)

Reply:
Plus, the `deployed-to-env` tag is only generated when an image is deployed to a specific environment - so this wouldn't work for new environments. Another reason is that the `deployed-to-production` tag is more likely to point to a working release (since folks frequently test things out on integration).